### DISCUSSION TITLE
Java Jersey Client 1.3 [Question]

### DISCUSSION THREAD

**Discussion by pontamilselvan** [featurehub-io/featurehub#548] [General]
Hi, 
I am using Java Jersey Client 1.3 and Core 1.3.  My question is, every time we need to create instance for JerseyClient and ClientRepository ? or can use as singleton objects?

**Comment by rvowles** [comment]
Hey there!

Generally you should only create one of these for your application, and then use the right kind of key to match. The only time you end up creating multiple clients and repositories will be for batch processing that is using multiple threading, where each thread has its own context for a strategy. Your average web server or Android app or single threaded client side app should only ever have one instance of each.
